cpp/tests/pagerank/pagerank_test.cpp

```diff
@@ -129,6 +130,7 @@ class Tests_Pagerank : public ::testing::TestWithParam<Pagerank_Usecase> {
     ASSERT_EQ(fclose(fpin), 0);
 
     //  Pagerank runs on CSC, so feed COOtoCSR the row/col backwards.
+    raft::handle_t handle;
```

Comment:
Do we need this

Reply:
Yea, PageRank asks for a `raft::handle_t const &handle`. Can't pass null there.

Reply:
then we should probably change line 147 and 154 to handle instead of G.handle[0]